### DISCUSSION TITLE
2. Build Lane, Road markings, Street Sign, Traffic Light detection pipeline from existing algorithms/models

### DISCUSSION THREAD

**Discussion by Polina106** [saed13/AI-PROJECT-SMART_RECORDING_PIPELINE#4] [General]
Models and datasets for the second subtask

**Comment by Micropot** [comment]
we can try to use the new model from Meta called SAM (Segment anything model). It can be use to segment by hand any element in images but also it can keep track of elements on a video. they're an example on their website with a VR mask.
